content/en/dora_metrics/deployments/_index.md

```diff
@@ -33,98 +33,39 @@ The DORA Metrics private beta is closed. Fill out the form below to be added to
-    --git-repository-url "https://github.com/organization/example-repository" \
-    --git-commit-sha 66adc9350f2cc9b250b69abddab733dd55e1a588
-```
-
-The deployment finish time is automatically set to now if `--finished-at` is not provided.
+## Calculating change lead time
 
-If the deployment CI job is running on the exact same Git revision that is being deployed, `git-repository-url` and `git-commit-sha` can be omitted and will be automatically inferred from the CI context.
-The `--skip-git` option can be provided to disable sending the repository URL and commit SHA. When this option is added, the Change Lead Time metric will not be available.
+For a single Git commit, the change lead time (CLT) is calculated as time from the creation of the commit to when the deployment including that commit was executed.
 
-[1]: https://www.npmjs.com/package/@datadog/datadog-ci
-[2]: https://app.datadoghq.com/organization-settings/api-keys
-{{% /tab %}}
-{{< /tabs >}}
+To calculate the change lead time for a deployment, Datadog runs [`git log`][6] between the deployment commit SHA and the previous deployment commit SHA to find all the commits being deployed. Then, it computes the average of the related change lead time values. Datadog doesn't store the actual content of files in your repository, only Git commit and tree objects.
 
-## Calculating Change Lead Time
+There are two requirements for calculating change lead time:
 
-For a single Git commit, the Change Lead Time (CLT) is calculated as time from the creation of the commit to when the deployment including that commit was executed.
-To calculate the Change Lead Time for a deployment, Datadog runs [`git log`][6] between the deployment commit SHA and the previous deployment commit SHA to find all the commits being deployed. Then, it computes the average of the related Change Lead Time values.
-Datadog doesn't store the actual content of files in your repository, only Git commit and tree objects.
-
-There are two requirements for calculating Change Lead Time:
 1. Both the Git repository URL and commit SHA are provided when sending deployment events.
 2. Your repository metadata is being [synchronized to Datadog](#synchronize-repository-metadata-to-datadog).
 
 ### Breakdown metrics
 
-Datadog also provides the following breakdown metrics, which represent the different stages since a commit is made until it is deployed.
+Datadog breaks down change lead time into the following metrics, which represent the different stages from commit creation to deployment.
 
-To compute these metrics, the PR associated with a commit must be identified, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR.
+To compute these metrics, there must be a pull request (PR) associated with a commit, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR.
```

Comment:
I think we should mention somewhere that the breakdown metrics (at least those related to PRs), are only created when the source of the repository metadata is GitHub

Reply:
```suggestion
These metrics are only computed when the source of the repository metadata is GitHub, and there must be a pull request (PR) associated with a commit, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR.
```